SPECS/systemd/systemd.spec

```diff
@@ -1186,6 +1188,9 @@ rm -f %{name}.lang
 # %autochangelog. So we need to continue manually maintaining the
 # changelog here.
 %changelog
+* Wed Mar 06 2024 Dan Streetman <ddstreet@microsoft.com> - 255-8
+- move /var/log/* files from filesystem to systemd package
```

Comment:
Just double-checking if any files need to be provided by the systemd package, since there isn't a specific /var/log/* diff here (other than the filesystem conflicts which deprecate specific files under /var/log that filesystem used to provide).

Reply:
Any chance containers will be writing to these folders and have issues? We don't have systemd in our containers by default

Reply:
i don't see why it would cause a problem?